doc/design/request_response_communication.md

```diff
@@ -40,34 +40,34 @@ In order to support asynchronous requests, a sequence ID should be part of each
 
 This is an overview of the untyped `Client` and `Server` classes.
```

Comment:
We said for now we won't have the design docs on the website, or?

Reply:
@budrus This is not yet included on the website and only solves the relative link debacle. Only things in `doc/website/` are on [iceoryx.io](https://iceoryx.io). My proposal is a three step approach: move the design docs to their respective module (`iceoryx_hoofs/doc/` or `iceoryx_posh/doc/`), revise them and then include them on [iceoryx.io](https://iceoryx.io).